drivers/gpu/drm/panel/panel-visionox-g2647fb105.c

```diff
@@ -1,40 +1,43 @@
 // SPDX-License-Identifier: GPL-2.0-only
```

Comment:
IMO it would be more appropriate to revert the old commit so that we can drop it nicely later.